requirements/xpu.txt

```diff
@@ -1,17 +1,24 @@
 # Common dependencies
 -r common.txt
 
-ray >= 2.9
+ray>=2.9
 cmake>=3.26
 ninja
 packaging
 setuptools-scm>=8
 setuptools>=75.8.0
 wheel
 jinja2
+datasets # for benchmark scripts
 
```

Comment:
I noticed that pandas also needed after #14036, please add it.

Reply:
`datasets` should have `pandas` dependency
![image](https://github.com/user-attachments/assets/730ad105-99c1-4ad3-8848-55d7899332c8)